modules/default.nix

```diff
@@ -105,10 +105,30 @@ inputs: {
     inherit (vimOptions) viAlias vimAlias withRuby withNodeJs withPython3;
     inherit extraLuaPackages extraPython3Packages;
   };
+
+  # Additional helper scripts for printing and displaying nvf configuration
+  # in your commandline.
+  printConfig = pkgs.writeShellScriptBin "print-nvf-config" ''
+    cat << EOF
+      ${vimOptions.builtLuaConfigRC}
+    EOF
```

Comment:
Could call the other script, and cat the result of that to save some space.

Reply:
I don't think that saves any space.

Reply:
Well this way, 2 files contain the built config, if we cat the first file instead of including the whole thing again, then only 1 file will.

Reply:
A derivation calling another derivation requires using runCommand - *not* a method that I prefer unless I have to.

Those scripts amount to roughly 55kbs in total, so imo the size difference is negligible.